financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinksignup/NetworkingLinkSignupViewModel.kt

```diff
@@ -73,6 +74,10 @@ internal class NetworkingLinkSignupViewModel @AssistedInject constructor(
     private val presentSheet: PresentSheet,
 ) : FinancialConnectionsViewModel<NetworkingLinkSignupState>(initialState, nativeAuthFlowCoordinator) {
 
+    private val pane: Pane by lazy {
+        determinePane(initialState.isInstantDebits)
+    }
```

Comment:
We might want to rename the viewmodel / screen so that it's not just `Networking`?  Maybe StripeJS uses better naming here?

Reply:
Also there's no changes on navigation when on Instant Debits? so we'd technically _navigate_ to `networking_link_signup`?

Reply:
Yep, that’s correct.